### PR TITLE
Make mobile work a little bit nicer

### PR DESCRIPTION
**Pull request by lukefrake**

### CODE REVIEW

File: _sass/_includes.scss [hunk 1]
```diff
@@ -13,6 +13,11 @@
 // Get our canvas setup
 @import "vendor/canvas";
 
+$screen-medium: 640px !default;
```
Comment (rafriki) [inline _sass/_includes.scss]: Why would these be in this file?

Comment (lukefrake) [inline _sass/_includes.scss]: Because foundation doesn't expose them simply. So the includes has become like the setup.

Where would you put it?

Comment (rafriki) [inline _sass/_includes.scss]: probably here https://github.com/spacebetween/spacebetween.github.io/blob/master/_sass/_variables.scss

Comment (lukefrake) [inline _sass/_includes.scss]: Touche

**Comment by rafriki** [comment]
:+1:

**Pull request merged**